### PR TITLE
Validate requests

### PR DESCRIPTION
**Pull request by rkondratowicz**
Using validator from previous PR

### CODE REVIEW

File: src/test/java/uk/gov/hmcts/reform/bulkscanccdeventhandler/handler/ExceptionRecordEventHandlerTest.java [hunk 1]
```diff
@@ -119,4 +121,17 @@ public void should_handle_transformation_result_with_warnings_when_errors_should
...
+    public void should_validate_request() {
+        // given
+        CaseCreationRequest req = SampleCaseCreationRequest.caseCreationRequest();
+        given(transformer.transform(req.exceptionRecord)).willReturn(okResult());
+
+        // when
+        handler.handle(req);
+
+        // then
+        verify(validator).validate(req);
```
Comment (doncem) [inline src/test/java/uk/gov/hmcts/reform/bulkscanccdeventhandler/handler/ExceptionRecordEventHandlerTest.java:135]: it's actually running for each test case. this new test case is not really needed for just verifying validator call

**Review by doncem — COMMENTED** [review]
new test case vs. verifying in existing?

File: src/test/java/uk/gov/hmcts/reform/bulkscanccdeventhandler/handler/ExceptionRecordEventHandlerTest.java (hunk 1 above)
Comment (rkondratowicz) [reply src/test/java/uk/gov/hmcts/reform/bulkscanccdeventhandler/handler/ExceptionRecordEventHandlerTest.java:135]: are you proposing to add this (`verify(validator).validate(req);`) to all existing tests?

Comment (doncem) [reply src/test/java/uk/gov/hmcts/reform/bulkscanccdeventhandler/handler/ExceptionRecordEventHandlerTest.java:135]: one is enough for me. not standing for all the team but i'm happy with one test

**Review by doncem — APPROVED** [review]

**Pull request merged**